### PR TITLE
Requirements.txt: Add it

### PR DESCRIPTION
**Pull request by RohanVB**
Closes: https://github.com/coala/coala-sublime/issues/17

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+coala_bears>=0.1.0.dev20160303173809
```
Comment (sils) [inline requirements.txt]: seriously? Can you upgrade that to 0.10? ;)

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+coala_bears>=0.10.0
```
Comment (meetmangukiya) [inline requirements.txt]: I think we Prefer new line at EOF not sure though

Comment (RohanVB) [inline requirements.txt]: done.

Comment (meetmangukiya) [inline requirements.txt]: @RohanVB review history showed that you went for some dev version of coala-bears, so I assume there might've been some reason for opting for that one. If it is, consider it a gentle reminder, if it is not, ignore ;)

Comment (RohanVB) [inline requirements.txt]: That was from: https://github.com/coala/coala-sublime/pull/19/files actually. I didn't check the version when I made the PR. I'm not sure what you mean - gentle reminder for?

**Comment by meetmangukiya** [comment]
Change commit to `Add requirements.txt` instead

**Comment by Mixih** [comment]
why circle if we don't test?

**Comment by RohanVB** [comment]
@Mixih that's what the cohesive editor support project tends to achieve :D

**Comment by Mixih** [comment]
ah, so just setting things up for the future? can we separate into a separate commit then please?

**Comment by RohanVB** [comment]
Closing because the plugin sucks and I'll be doing https://github.com/coala/coala-sublime/issues/36 anyways.

**Pull request closed without merging**